scripts/clean_vendor.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Removes unncessary files from the vendor directories
+#
+# A aist for files to be removed is used instead of excluding files.
+# The reason is that this that this makes the setup simpler and prevents
+# from removing files by accident
+#
+# In general it should always be removed manually which files were removed.
+# For example some projects like stretchr/testify have two LICENSE files
+# with different names, where it is ok that one is removed.
+#
+# We keep the CHANGELOG in as this makes it easy visible when updating dependencies
+# on what has changed.
+
+# In general the following files should b kept:
+# * .go
+# * LICENSE* CHANGELOG* and PATENT*
```

Comment:
@urso That here is more a general remark. I have the LICENSES below as there was one project using it. I would only add "exception" if they occur ...